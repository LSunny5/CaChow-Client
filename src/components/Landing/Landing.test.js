import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import LandingPage from './Landing';

describe(`Landing Page component`, () => {
	it('renders the Landing Page by default', () => {
		const wrapper = shallow(<LandingPage />);
		expect(toJson(wrapper)).toMatchSnapshot();
	});
});