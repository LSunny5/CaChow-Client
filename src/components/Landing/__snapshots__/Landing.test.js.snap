// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing Page component renders the Landing Page by default 1`] = `
<section
  className="LandingContent"
>
  <div
    className="LandingTop"
  >
    <img
      alt="Cachow Logo"
      className="LandingImage"
      src="images/LogoResized.png"
    />
    <h1
      className="LandingHeading"
    >
      CaChow!
    </h1>
  </div>
  <div
    className="LandingBottom"
  >
    <div
      className="LandingDescription"
    >
      Welcome to the site that keeps all your menus in one place! Instead of having to navigate to several different websites to view menus and restaurant information, keep all your menus in one place!
    </div>
    <div
      className="ButtonTextBox"
    >
      <div
        className="ButtonText"
      >
        Let's Begin
      </div>
      <div
        className="ButtonWrapper"
      >
        <div
          className="ButtonArrow"
        />
        <NavLink
          to="/login"
        >
          <div
            className="ButtonCircleBorder"
          />
          <div
            className="ButtonCircleCover"
          >
            <div
              className="SmallCircle"
            />
          </div>
        </NavLink>
      </div>
    </div>
  </div>
</section>
`;
